cachi2/core/package_managers/pip.py

```diff
@@ -920,7 +934,7 @@ def from_requirements_and_options(
         :param list options: list of strings of global options
         :return: new instance of PipRequirementsFile
         """
-        new_instance = cls("")
+        new_instance = cls(RootedPath("/"))
```

Comment:
Don't really like this solution, but it is probably not much worse that the previous one. This is a workaround for having a "in-memory" PipRequirementsFile that is used for replacing the paths of file/vcs dependencies, so the actual `self.file_path` is never really called.

Maybe we could try to refactor an actual InMemoryPipRequirementsFile and remove the need from a file path from it.